components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/exception/session/mgt/SessionManagementClientException.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.application.authentication.framework.exception.session.mgt;
+
+import org.wso2.carbon.identity.application.authentication.framework.util.SessionMgtConstants;
+
+/**
+ * This class is used to define the client side errors which needs to be handled.
+ */
+public class SessionManagementClientException extends SessionManagementException {
+
+    public SessionManagementClientException(SessionMgtConstants.ErrorMessages error, String
+            description, Throwable cause) {
+
+        super(error, description, cause);
+
```

Comment:
Remove extra lines. Do the same for all the instances.

Reply:
done